etc/stripGroupTags.py

```diff
@@ -0,0 +1,15 @@
+import sys
+import re
+
+blacklist = [
+    "Data I/O endpoints",
+    "Security and access endpoints",
+    "System information endpoints"
+]
```

Comment:
Maintaining this list will likely be a painpoint moving forward. Can you block the parent `Tags` field and it's contents altogether? Might have to utilize proper yaml parsing, but the more hands-off it is, the better.

Reply:
I initially tried removing all tags, but this causes the API stubs to be thrown in a generic interface, instead of an interface such as `Tasks`. I wasn't able to find a way to group apis together without using the tag system either.

Reply:
So strange.. I would expect from your former error:
```
api/api_security_and_access_endpoints.gen.go:618:6: ApiGetAuthorizationsRequest redeclared in this block
        api/api_authorizations.gen.go:259:6: other declaration of ApiGetAuthorizationsRequest
```
that it didn't generate interfaces nor structs that implement them, but flat functions grouped into named files. Seems if they use tags to create an interface, they should preserve that thought and make functions (like `ApiGetAuthorizationsRequest` be memeber functions of some tag-named-ish struct. What uses the generated interface(s) and would it still work despite it being too coupled?

Reply:
The generated interfaces are used all throughout the codebase, not sure what you mean with `would it still work despite it being too coupled?`